Packs/Hey/Scripts/Hey/Hey.yml

```
@@ -0,0 +1,93 @@
+commonfields:
+  id: Hey
+  version: -1
+fromversion: 6.0.0
+comment: Use rakyll/hey to test a web application with a load of requests.
+name: Hey
+script: ''
+type: python
+tags: []
+enabled: true
+args:
+- name: url
+  required: true
+  description: URL to query.
+- name: requests_number
+  description: Number of requests to run. Default is 200.
+- name: timeout
+  description: Timeout for each request in seconds. Default is 20, use 0 for infinite.
+- name: concurrency
+  description: Number of workers to run concurrently.
+- name: duration
+  description: Duration of application to send requests (in seconds).
+- name: method
+  auto: PREDEFINED
+  predefined:
+  - GET
+  - POST
+  - PUT
+  - DELETE
+  - HEAD
+  - OPTIONS
+  description: HTTP method.
+- name: headers
+  description: Custom HTTP header. Comma separated list
+    of "key=value". e.g. Type=IP,Size=100.
```

Let's use real headers for the example:
```suggestion
    of "key=value". e.g. User-Agent=curl/7.54.0,Accept=*/*
```